dashboard/static/css/grades.css

```diff
@@ -1,7 +1,3 @@
-body {
-    font: 10px sans-serif;
```

Comment:
I would make sure removing would have little impact the view

Reply:
Thanks, some views might be impacted. If you want smaller fonts in some cases just target a class or something in the content rather than the body.

Reply:
You can even edit this PR if you like and add the changes here!

Reply:
Too late I created the issue  and fixed it . once you merge this change I will merge mine. https://github.com/tl-its-umich-edu/student-dashboard-django/pull/98